tests/system/marathon_common_tests.py

```diff
@@ -642,9 +642,13 @@ def test_pinned_task_does_not_scale_to_unpinned_host():
 
     app_def = apps.sleep_app()
     app_id = app_def['id']
-    app_def['cpus'] = 3.5
+
     host = common.ip_other_than_mom()
     print('Constraint set to host: {}'.format(host))
+    # the size of cpus is designed to be greater than 1/2 of a node
```

Comment:
What if a node has only 1 cpu? Should it be something like `max(0.6, cores - 0.5)` instead?

Reply:
it's a good catch... however if we start using 1 core vms I quit :)